tests/formspree_test_case.py

```diff
@@ -0,0 +1,34 @@
+
+import os
+import fakeredis
+
+from flask.ext.testing import TestCase
+import formspree
+from formspree import create_app
+from formspree import settings
+from formspree.app import DB, REDIS
+
+
+class FormspreeTestCase(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        ''' configures app for testing '''
+        formspree.app.get_redis = lambda : fakeredis.FakeStrictRedis()
```

Comment:
(continuing from my last comment...) Because what you are doing here needs to be undo manually on the tearDown, otherwise other tests that are expecting `get_redis` to return something else would crash.